openreview/conference/builder.py

```diff
@@ -1532,22 +1526,29 @@ def is_release_authors(is_note_accepted):
                     paper_status = 'accepted' if note_accepted else 'rejected',
                     anonymous=False
                 )
-            #add venue_id if note accepted
+            #add venue_id if note accepted and build venue_heading_map
             venue = self.short_name
+            decision_option = decision_note.content['decision'] if decision_note else None
```

Comment:
instead of build the map here when you are iterating all the notes where some decisions are duplicated, why don't you build it before calling set_homepage_decisions:

```
for decision in decisionOptions:
     venue_heading_map[self.decisionToVenue(decision)] = decision_heading_map[decision]
```

We can define a new function `decisionToVenue` that convert the decision string to a venue value.